packages/editor-ui/src/components/Node.vue

```diff
@@ -214,6 +243,75 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		isDragActiveTriggerNode: {
+			handler(isDragActiveTriggerNode) {
+				if (this.workflowRunning && this.isTriggerNode && this.isSingleActiveTriggerNode && !this.isTriggerNodeTooltipEmpty && !this.isNodeDisabled && !this.hasIssues) {
+					this.showWebhookNodeTooltip = !isDragActiveTriggerNode;
+				}
+			},
+		},
+		nodePosition: {
+			handler() {
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						if (this.workflowRunning && this.isSingleActiveTriggerNode) {
+							this.showWebhookNodeTooltip = true;
+						}
+					}, 200);
+				}
+			},
+			immediate: true,
+			deep: true,
+		},
+		nodeOffsetPosition: {
+			handler() {
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						if (this.workflowRunning) {
+							this.showWebhookNodeTooltip = true;
+						}
+					}, 200);
+				}
+			},
+			immediate: false,
+			deep: true,
+		},
+		workflowRunning: {
+			handler(isWorkflowRunning) {
+				if (isWorkflowRunning && this.isTriggerNode && this.isSingleActiveTriggerNode && !this.isTriggerNodeTooltipEmpty && !this.isNodeDisabled) {
+					setTimeout(() => {
+						if (!this.isNodeDisabled && !this.hasIssues && this.isSingleActiveTriggerNode) {
+							this.showWebhookNodeTooltip = this.workflowRunning;
+						}
+					}, 2500);
+				} else {
+					this.showWebhookNodeTooltip = false;
+				}
+			},
+			immediate: true,
+		},
+		isNodeDisabled: {
```

Comment:
as discussed we should try to simplify these

Reply:
Fixed.